src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -196,14 +196,19 @@ await requestExecutor.ExecuteAsync(_redirectNode, null, context, command, should
                         tcpInfo = await LegacyTryGetTcpInfo(requestExecutor, context, token).ConfigureAwait(false);
                     }
                 }
+                
+                string chosenUrl; 
+                (_tcpClient, chosenUrl) = await TcpUtils.ConnectAsyncWithPriority(tcpInfo, 
+                    requestExecutor.DefaultTimeout.GetValueOrDefault(RequestExecutor.GlobalHttpClientTimeout), 
```

Comment:
we should not use `GlobalHttpClientTimeout`. It is set to 12h.

Reply:
We should use default TcpClient timeouts then?

Reply:
Thought about it, but at line 201 we have not yet created TcpClient.
Perhaps we should add the timeouts to client Conventions? 
The DefaultTimeout is there I think, is there a reason we shouldn't have RequestTimeout and ResponseTimeout there?